Newfeatures

david63011 opened the pull request:
I added my HTML and CSS files for the first part of my website

Danieldotcomcoder reviewed (commented):
Hi @david63011 , 


While you made a great effort in this project, unfortunately, I cannot proceed to review your code.

## Invalid Code Review Request

You have submitted a project, that doesn't include a professional Readme File , Kindly refer to this [template](https://github.com/microverseinc/readme-template/blob/master/README.md) 👍 👍  to add a good Readme File 😄 😄 then submit again, Thank you 👍 

Your Code Review Request will be marked as invalid in your Dashboard, so please submit a new one once you are ready 🙏

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

--------

_Invalid Code Review Request **does not count** into the code reviews limit._

tugsem commented on index2.html:
```diff
@@ -0,0 +1,42 @@
+    <p class="welcome">Welcome</p>
+    <img class="union" src="Union.png">
+    </div>
+   </div>
```
- [ ] In order to make your HTML semantic and organized, please use `<header>` tag. Div is not prefferable when you are creating a wrapper such as header, main, article, section, footer, form etc... If you can't find a semantic tag for your purpose, you can use `div` then. Also for `img` tags, kindly add alt attribute to make it more accessible by screen-readers. You can define the purpose of the image inside it. Such as logo, menuBtn... and please don't forget to add relative path in your src attributes like this `./Union.png` . Otherwise, it's not gonna show off in other browsers.

tugsem commented on index2.html:
```diff
@@ -0,0 +1,42 @@
+        <p class="info5">a project you need coded</p>
+        <p class="info6"> don’t hestiate to contact me.</p>
+      </div>
+    </div>
```
- [ ] Kindly change this div wrapper to something more semantical like `<section>` to make your HTML more professional.

tugsem commented on index2.html:
```diff
@@ -0,0 +1,42 @@
+        <p class="info6"> don’t hestiate to contact me.</p>
+      </div>
+    </div>
+    <img class="icons" src="social-icons.png">
```
- [ ] Kindly add icons by creating list elements with anchor tags and be careful about spacings between them. You can use flex to arrange them. Do not forget to link them to your accounts as much as possible. This portfolio will likely be your showcase for your upcoming projects in Microverse.

tugsem commented on style2.css:
```diff
@@ -0,0 +1,145 @@
+  margin-left: 230px;
+}
+
+.flex {
+  max-width: 375px;
+  flex-wrap: wrap;
+}
```
- [ ] It's nice to give this classname to flex container, but in order to make your code DRY, please use this class for every flex containers and add `display: flex;`  in it so that the name can have more meaning.

tugsem commented on style2.css:
```diff
@@ -0,0 +1,145 @@
+}
+
+.info6 {
+  font-family: Arial, Helvetica, sans-serif;
+  font-style: normal;
```
- [ ] Kindly remove this lines from your CSS since you are repeating them too much and this is not a good practice for DRY principle. You don't need to declare font-style as normal since it's default. Also when you want every item have the same feature then you can use this selector `*{}` at the beginning.

tugsem commented on style2.css:
```diff
@@ -0,0 +1,145 @@
+  line-height: 0.5;
+  color: #172b4d;
+  margin-left: 35px;
+}
+
+.info4 {
+  font-family: Arial, Helvetica, sans-serif;
+  font-style: normal;
+  font-weight: 400;
+  font-size: 22px;
+  line-height: 0.5;
+  color: #172b4d;
+  margin-left: 35px;
+}
+
+.info5 {
+  font-family: Arial, Helvetica, sans-serif;
+  font-style: normal;
+  font-weight: 400;
+  font-size: 22px;
+  line-height: 0.5;
+  color: #172b4d;
+  margin-left: 35px;
+}
+
+.info6 {
+  font-family: Arial, Helvetica, sans-serif;
+  font-style: normal;
+  font-weight: 400;
+  font-size: 22px;
+  line-height: 0.5;
+  color: #172b4d;
+  margin-left: 35px;
+}
+
+.icons {
+  position: absolute;
+  top: 670px;
+  left: 90px;
+}
```
- [ ] Your design layout doesn't fits well on mobile screens. Kindly fix that extra spacing on the right and bottom and provide the background picture as the figma template here:

![image](https://user-images.githubusercontent.com/91486791/185479195-708e5cbd-b602-4ad0-8844-ccc246f49aab.png)
![image](https://user-images.githubusercontent.com/91486791/185480330-78e351e4-d0a4-4df9-a47b-4c0084a4fcea.png)

david63011 replied:
sorry about this one I don't understand the background picture is working for me

david63011 replied:
I guess I need to send you the photos sorry about that I will them when I resend it tonight

tugsem reviewed (changes requested):
Hi @david63011 , 

Good job so far! 👏 
There are some issues that you still need to work on to go to the next project but you are almost there!
![ Alt Text](https://media.giphy.com/media/KyBesf8kNT6s88SEgb/giphy.gif)
## Highlights
✅ Correct Gitflow use,
✅ All linters passing
✅ Descriptive readme file

## Required Changes ♻️
_Check the comments under the review._

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

- [ ] I suggest using more descriptive branch names rather than naming it as newfeatures. 🎯 

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

tugsem commented on index2.html:
```diff
@@ -0,0 +1,39 @@
+    <div class="flex">
+      <img src="background.png" alt="green background" />
+      <div class="greeting">
+        <p class="hi">Hey there. I'm</p>
+        <p class="first-name">Mariane</p>
+        <p class="job">I am a software</p>
+        <p class="job-info">Developer</p>
```
- [ ] Please use `h1` tag for this title here since this way will be semantically correct and the screen readers and other technologies will thank you.

tugsem commented on index2.html:
```diff
@@ -0,0 +1,39 @@
+    </div>
+    <div class="bottom-section">
+      <p class="welcome">Welcome</p>
+      <img class="union" src="Union.png" alt="union photo" />
```
- [ ] Please use button tag for hamburger icon since you will add functionality to it in the upcoming projects

tugsem commented on index2.html:
```diff
@@ -0,0 +1,39 @@
+      <img class="cell-info" src="Light.png" alt="cell phone info" />
+    </div>
+    <div class="bottom-section">
+      <p class="welcome">Welcome</p>
```
- [ ] Kindly add link (a tag) to your logo here. This is for three reasons: convention, semantics, and user experience(UX).

david63011 commented:
I made many changes I added an a tag to my welcome, I changes the p tags to h1 tags, I deleted some unnecessary things from my CSS, I changed the name of one of my containers, and I changed the social media icons to individual icons and I changed three to have a tag with my social media information, I added section and header tags,.

I didn't change the hamburger icon to a button because I don't know how to do that without javascript but I will in the future!
And the background was already there but I didn't send you the photos. So I hope everything now is ok because I need to continue the project thanks for your help.

tresorsawasawa reviewed (commented):
# STATUS: INVALID CODE REVIEW REQUEST 🛑 🛑

Hi @david63011 :wave: :wave: 
While you made a great effort in this project, unfortunately, I cannot proceed to review your code.

## Invalid Code Review Request 🛑 🛑

You have submitted a project that has "merging conflicts" with the default branch that needs to be fixed and where linters don't check. Make sure you fix them before requesting for the new request review.

Your Code Review Request will be marked as invalid in your Dashboard, so please submit a new one once you are ready 🙏

---
Cheers and Happy coding!👏👏👏

_Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please ping me [@tresorsawasawa](https://github.com/tresorsawasawa) when you comment so I can receive the notification or use slack with the same name._

---
_Invalid Code Review Request **Does not count** into the code reviews limit._
